app/backend/test/teet/test/utils.clj

```diff
@@ -0,0 +1,83 @@
+(ns teet.test.utils
+  (:require [datomic.client.api :as d]
+            [teet.db-api.db-api-handlers :as db-api-handlers]
+            [teet.environment :as environment]
+            [teet.log :as log]
+            [teet.user.user-db :as user-db]))
+
+;;
+;; Test fixtures
+;;
+
+(defn with-environment [f]
+  (log/info "Loading local config.")
+  (environment/load-local-config!)
+  (f)
+  (log/info "Reloading local config.")
+  (environment/load-local-config!))
+
+(defn with-db [f]
+  (log/redirect-ion-casts! :stderr)
+  (let [test-db-name (str "test-db-" (System/currentTimeMillis))]
+    (log/info "Creating database " test-db-name)
+    (swap! environment/config assoc-in [:datomic :db-name] test-db-name)
+    (d/create-database (d/client (environment/config-value :datomic :client)) {:db-name test-db-name})
+    (f)
```

Comment:
I think this should be wrapped in exception handling to avoid leaving databases.

```clojure
(try 
  (f)
  (finally
    (d/delete...))
```

Reply:
I tested this, and the test framework seems to catch the exceptions and allow the fixtures to finish without a `try` block:
```
INFO 2020-04-07 10:09:46.777 [] teet.test.utils:26 - Deleting database  test-db-1586243375591
INFO 2020-04-07 10:09:47.188 [] teet.test.utils:16 - Reloading local config.
[...]
ERROR in teet.system-test/throw-stuff (system_test.clj:14)
Uncaught exception, not in assertion.
Exception: clojure.lang.ExceptionInfo: thrown
[...]
```